bundles/org.openhab.binding.sensibo/src/main/resources/OH-INF/thing/sensibosky.xml

```diff
@@ -10,7 +10,7 @@
 		</supported-bridge-type-refs>
 
 		<label>HVAC controller</label>
-
+		<semantic-equipment-tag>Thermostat</semantic-equipment-tag>
```

Comment:
A Thermostat should have a setpoint, this device more looks like a `HVAC`

Reply:
Changed.